.github/stale.yml

```diff
@@ -0,0 +1,19 @@
+daysUntilStale: 90
+daysUntilClose: 90
+
+exemptLabels:
+- type/bug
+- type/cleanup
+- type/feature
+- type/infrastructure
+- critical
+- security
+exemptMilestones: true
+exemptAssignees: true
+
+staleLabel: wontfix
```

Comment:
We don't have that label (yet) - is that a problem? Also, what exactly does this do? Does the bot apply this when marking as stale? Do we need that?

Reply:
The stale label is added to issues/PRs when the issue is marked as stale. It is needed as it separates the period of marking as stale from closing the issue.

Reply:
So then issues will be labeled as "wontfix" before they are closed? IMO, "stale" sounds like the more logical name then?